src/userportal.config

```diff
@@ -1,5 +1,5 @@
 {
   "debug": true,
-  "applicationContext": "/ovirt-engine",
-  "applicationURL": "/ovirt-engine/web-ui"
+  "applicationContext": "",
```

Comment:
Did you try in production environment (SSO, .war deployment) ?
I think this will not work there.